entries/n/nerdwallet.com.json

```
@@ -0,0 +1,16 @@
+{
+  "NerdWallet": {
+    "domain": "nerdwallet.com",
+    "contact": {
+      "facebook": "NerdWallet",
+      "twitter": "NerdWallet"
+    },
+    "regions": [
+      "us",
+      "ca"
```

https://support.nerdwallet.com/hc/en-us/articles/115001681666
Accounts are available for US residents only.
```suggestion
      "us"
```